components/console/src/core/infrastructure/mongo/mappers/mongo-organization-avatar-mapper.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file Organization Avatar Mapper
+ * @description Provides mapping functionality between domain entities and MongoDB documents
+ * for organization avatars, following the Domain-Driven Design pattern.
+ */
+
+import { OrganizationAvatarEntity } from '@/core/domain/entities/organization-avatar-entity'
+import { OrganizationAvatarDocument } from '../models/models/organization-avatar'
+
+/**
+ * OrganizationAvatarMapper
+ * @class
+ * @description Maps between domain entities and MongoDB documents for organization avatars.
+ * This mapper ensures proper separation between domain and infrastructure layers
+ * by providing clean transformation methods in both directions.
+ */
+export class OrganizationAvatarMapper {
+  /**
+   * Converts a MongoDB document to a domain entity
+   * @param doc - The MongoDB document containing organization avatar data
+   * @returns A clean domain entity with only the properties defined in the domain model
+   */
+  static toEntity(doc: OrganizationAvatarDocument): OrganizationAvatarEntity {
```

Comment:
Super nitpik:
I know mongoose uses the "Document" terminology, but should we try to maintain the "Dto" naming convention, or is it fine to keep the "Document"?

Example: OrganizationAvatarDocument -> MongoOrganizationAvatarDto